### PR TITLE
Add download/installation instructions for Fedora

### PR DESCRIPTION
**Pull request by uklotzde**
The corresponding `<div>` sections are equal and redundant. There is no distinction between a stable and a pre-release version within the RPM Fusion repositories.

Currently we are distributing a pre-release version for F29 that has been released prior to the expected Mixxx 2.2 release. This is an exception, usually only stable versions will be distributed.

### CODE REVIEW

File: pages/download.html [hunk 1]
```diff
@@ -152,7 +154,36 @@ <h2><img src="{% static '/static/images/download_ubuntu.png' %}" class="feature_
...
+  </div>
+
+  <div class="halfbox_right darkborder fedorabox">
+    <h2><img src="{% static '/static/images/download_fedora.png' %}" class="feature_icon">{{ fedora }}</h2>
+
+    <p>{% blocktrans %}
+        An <a href="https://admin.rpmfusion.org/pkgdb/package/free/mixxx/">RPM package</a>
+        for installation is available in the
+        <a href="https://rpmfusion.org">RPM Fusion</a> repositories.
+      Please refer to their instructions on how to enable the repositories on your system.
```
Comment (Be-ing) [inline pages/download.html]:
```suggestion
      Please refer to their <a href="https://rpmfusion.org/Configuration">instructions</a> on how to enable the repositories on your system.
```

Comment (Be-ing) [reply pages/download.html]: Add: "Mixxx only requires enabling the free repository; the nonfree repository is not necessary for Mixxx."

File: pages/download.html [hunk 2]
```diff
@@ -244,7 +275,36 @@ <h2><img src="{% static '/static/images/download_ubuntu.png' %}" class="feature_
...
```
Comment (Be-ing) [inline pages/download.html]: Above comments apply here too

**Comment by uklotzde** [comment]
Anything else to do? I would like to merge this before we officially announce 2.2.0.

Fedora has become one of the best supported platforms since I was accepted to maintain the RPM Fusion package - full-featured, tested, optimized, and always up to date. Users should know about this ;)

**Comment by Be-ing** [comment]
LGTM, thanks for adding this to the website and maintaining the package.

**Pull request merged**